e2e/test.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs')
+const path = require('path')
+
+const shell = require('shelljs')
+
+const chimiBin = path.resolve(__dirname, '..', 'bin', 'bin.js')
+const fixturesRoot = path.join(__dirname, 'fixtures')
+
+const fixturesDirs = fs
+  .readdirSync(fixturesRoot)
+  .map(x => path.join(fixturesRoot, x))
+  .filter(x => fs.lstatSync(x).isDirectory())
+
+describe('e2e tests', () => {
+  for (let fixtureDir of fixturesDirs) {
+    const relativeFixtureDir = path.relative(fixturesRoot, fixtureDir)
+    it(`should run ${relativeFixtureDir} as expected`, () => {
```

Comment:
I like that the directory name is the description, but let's take it an step further and make it the only description of the test.

```js
it(relativeFixtureDir.replace(/^[0-9]*/, '').replace(/-/g, ' '))
```

Reply:
Not sure about this. The way it is now has more readable code and it's more straightforward to know which directory made the test fail. I wouldn't trade that for a cleaner test output. Any advantage I'm missing?

Reply:
> Any advantage I'm missing?

I'd say only

> a cleaner test output

:joy: